lib/stack_master/stack_differ.rb

```diff
@@ -33,6 +40,9 @@ def output_diff
       if @current_stack
         text_diff('Stack', current_template, proposed_template, context: 7, include_diff_info: true)
         text_diff('Parameters', current_parameters, proposed_parameters)
+        unless noecho_keys.empty?
+          StackMaster.stdout.puts " * can not tell if NoEcho parameters are different, assuming not."
```

Comment:
I think we need to assume they _are_ different.

* Worst case if we assume they are different and they're not: stack update happens, stops with "nothing to update".
* Worst case if we assume they are not different and they are: stack not updated with changed params.